src/command_modules/azure-cli-cdn/azure/cli/command_modules/cdn/_help.py

```diff
@@ -0,0 +1,160 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=unused-import
+from azure.cli.core.help_files import helps
+
+# pylint: disable=line-too-long
+helps['cdn'] = """
+    type: group
+    short-summary: Manage Azure Content Delivery Networks (CDN)
+"""
+
+helps['cdn profile create'] = """
+    type: command
+    examples:
+        - name: Create a CDN profile using Verizon premium CDN
+          text: >
+            az cdn profile create -g group -n profile --sku Premium_Verizon
+"""
+
+helps['cdn profile update'] = """
+    type: command
+    short-summary: Updates a CDN profile with a profile name under the specified resource group.
+"""
+
+helps['cdn profile delete'] = """
+    type: command
+    examples:
+        - name: Delete a CDN profile
+          text: >
+            az cdn profile create -g group -n profile
+"""
+
+helps['cdn profile list'] = """
+    type: command
+    short-summary: List your Azure Content Delivery Network (CDN) profiles
+    examples:
+        - name: List CDN profiles
+          text: >
+            az cdn profile list
+"""
+
+helps['cdn endpoint create'] = """
+    type: command
+    examples:
+        - name: Create an endpoint with a custom domain origin
+          text: >
+            az cdn endpoint create -g group -n endpoint --profile-name profile \\
+                --origin www.example.com
+        - name: Create an endpoint with a custom domain origin with HTTP and HTTPS ports
+          text: >
+            az cdn endpoint create -g group -n endpoint --profile-name profile \\
+                --origin www.example.com 88 4444
+        - name: Create an endpoint with a custom domain with compression and only HTTPS
+          text: >
+            az cdn endpoint create -g group -n endpoint --profile-name profile \\
+                --origin www.example.com --no-http --enable-compression
```

Comment:
The order would be consistent--your PR is defining the order. 😁 

My argument is that this is bizarre:
`--origin www.foo.com 80 521 --no-http # have to specify SOMETHING for http so I can set custom https`

And this makes more sense:
`--origin www.foo.com 521 --no-http`

HTTP will have a default of 80 but it doesn't matter because I'm specifying --no-http anyways.

If I wanted a custom http port I would need to specify the https port too, but then it would make no sense anyways to use --no-http (though it would still work).